### PR TITLE
Support oversea torritories in analyser_merge_milestone_FR #2282

### PR DESCRIPTION
**Pull request by frodrigo**
@didier2020 I made it works. But you need to review the content of the `is_milestone()` function.

### CODE REVIEW

File: analysers/analyser_merge_milestone_FR.py
Comment (didier2020) [inline analysers/analyser_merge_milestone_FR.py:1]: as dep can be on 3 digit 
to Filter only real milestone (not logical as DRD, FRG, CS etc...)
change 
            return row['nom_plo'][2:4] == 'PR'
by
            return (row['nom_plo'][2:4] == 'PR' or row['nom_plo'][2:5]  == 'PR')

Comment (didier2020) [reply analysers/analyser_merge_milestone_FR.py:1]: guadeloupe is not in the data it's guyanne FR-GF with resource e9288a97-faf1-4638-b3df-571ceae401f8

Comment (frodrigo) [reply analysers/analyser_merge_milestone_FR.py:1]: Both fixed.

**Comment by frodrigo** [comment]
I also see a N1 in FR-PM, but excluded by is_milestone. Is it expected ?

**Comment by didier2020** [comment]
is_milestone() function is ok.
#I also see a N1 in FR-PM, but excluded by is_milestone. Is it expected ?
there is no wrong filter on "route" and "nom_plo". N1 is in "lib_rte"

**Comment by frodrigo** [comment]
Ok, thank you, merged.

**Pull request merged**